cmake/DockerConfig.cmake

```diff
@@ -20,6 +20,35 @@ add_custom_target(
     docker
     COMMAND ${CMAKE_SOURCE_DIR}/docker/DockerBuild.sh 1000 1000 ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH} minificppsource ${CMAKE_SOURCE_DIR}
     WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/docker/)
+    
+
+add_custom_target(
```

Comment:
Do you think it might be better to just have these as their own Makefile?  If we did that, folks could get all these builds with just Docker whereas they would also need to go through the cmake'ing and need its dependencies when it's just formulating the targets.

Reply:
That was a consideration, but then make all in the build process would not be able to make everything. We'd have to pull up a parent Make to launch cmake produced targets, which seems less than ideal. Further, even not withstanding these considerations, the users would have two places to call make as a result.

Reply:
sounds reasonable